test/user_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require 'pry'
+
+class UserTest < Minitest::Test
+
+  def setup
+    @sal = User.new("Sal")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    @ali = User.new("Ali")
+  end
+
+  def test_user_exists
+    assert_instance_of User, @sal
+    assert_instance_of User, @ali
+  end
```

Comment:
This is good. Don't really have to check both objects, since you set them up the same way. If it worked for Sal, it'll work for Ali.